Dockerfile.latest

```diff
@@ -5,26 +5,24 @@ ENV IPFS_MONITORING=1
 ENV IPFS_PATH=/root/.jsipfs
 ENV BUILD_DEPS='libnspr4 libnspr4-dev libnss3'
 
-RUN apk add --no-cache git python3 build-base
-
-# Hopefully remove when https://github.com/node-webrtc/node-webrtc/pull/694 is merged
-RUN npm install -g ipfs@"$IPFS_VERSION"
-
-# Make the image a bit smaller
-RUN npm cache clear --force
-RUN apk del build-base python3 git
-
-# Configure jsipfs
-RUN jsipfs init
-
-RUN jsipfs version
-
-# Allow connections from any host
-RUN sed -i.bak "s/127.0.0.1/0.0.0.0/g" $IPFS_PATH/config
+RUN \
+    apk add --no-cache catatonit bash procps nodejs npm \
```

Comment:
The `node:16-alpine` container comes with node & npm so there shouldn't be a need to install more copies.

That said, this should be running on LTS node and not node 16 now.

Reply:
It's not based on node:16-alpine anymore because then you can never clean node and npm out of the final image.  That's why the last one was almost a gigabyte.  You cannot "shrink" or "remove", in later layers, things that were added in previous layers. the only way to not have npm in the target image is to use it as a builder layer and copy out, OR do as I have done here, add and remove it in a single layer. This is the correct approach because we want to keep nodejs, otherwise we would use node:alpine as a builder and just copy out our targets.
I will check on the versions this pulls.  maybe we can rebase to alpine 17 and have the lts if this doesn't already.

Reply:
edited my comment to reflect that we only want to scrub out npm.  obvs we need to keep nodejs in there.

Reply:
oh lol i see what you mean now sorry.  it uses node 18.12.1 this way, which is lts.